.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+# Description:
+<!-- Summary of change, including the issue to be addressed. -->
+
+## Type of change:
+<!-- Put an `x` in the box that applies. -->
+- [ ] Bug fix.
+- [ ] New feature.
+- [ ] Documentation update.
+
+## Checklist for the reviewer:
+<!-- Put an `x` in the boxes that apply. These can be filled by reviewer after the PR is created. -->
+
+This checklist should be used as a help for the reviewer.
+
+- [ ] Is the change limited to one issue?
```

Comment:
From the following lists (https://www.evoketechnologies.com/blog/code-review-checklist-perform-effective-code-reviews/ https://dev.to/alexomeyer/code-review-a-comprehensive-checklist-5gnm) we can maybe add a few more categories: 

Are we following internal coding style?
Is there clear code duplication with other sections?
Are the variables clearly named (overlaps with readability a bit but I think being specific can help)
Is one class developing too many responsibilities?

Reply:
I agree that the list can be made longer, but we agreed to start with 5 points and see how they proceed.
Internal coding style should be taken care of by all the stuff in pre-commit I think. Variables clearly names I can specify in the readability. For the other two I think it is better to start with few points and then I suggest you bring up these points at the next retrospective.

Reply:
I think this makes sense, I am approving all of them. Much better, as you say, to start with a few points and add up as-needed.